src/alchemtest/amber/simplesolvated/descr.rst

```diff
@@ -0,0 +1,22 @@
+Amber: Small molecule thermodynamic integration free energy difference in water. This example uses ligands 17124-1 to 18637-1 from J. Am. Chem. Soc. 2015, 137, 2695−2703. 
+=========================
+
+Notes
+-----
+Data Set Characteristics:
+    :Number of Legs: 2 (charge, vdw)
+    :Number of Windows: 5 for charge, 12 for vdw
+    :Length of Windows: 1ns
+    :System Size: 5979 atoms
+    :Temperature: 300 K
+    :Pressure: 1 bar
+    :Alchemical Pathway: (charge + vdw) in ligand 1 --> (charge + vdw) in ligand 2, charge and vdw are running in parellel, soft core is used in vdw
+    :Experimental Free Energy difference: N/A 
+
+    :Missing Values: None
+    :Date: Oct 2017
+    :Donor: Silicon Therapeutics 
```

Comment:
**IMPORTANT**: add a line
```reST
:License: `NAME <URL>`_
```
that specifies the license that the dataset is released under. See [Contributing: Licenses](https://github.com/alchemistry/alchemtest/wiki/contributing#licensing). Specifically, pick a license from the list [recommended conformant licenses](http://opendefinition.org/licenses/#recommended-conformant-licenses) (with CC0 or CC-BY preferred). You might have to check with your legal department. 

In PR #7 I updated the GMX dataset and you can have a look at it to see what I did there.

Reply:
Yes, will consult with the company and should get it back soon.